infra/cmake/packages/TensorFlowLite-1.13.1/Lite/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # NOTE The followings SHOULD be defined before using this CMakeLists.txt
 #
 #  'TensorFlowSource_DIR' variable
-#  'FlatBuffersSource_DIR' variable
+#  'flatbuffers-2.0' target
```

Comment:
(optional) how about adding a note like
"TensorFlow 1.13.1 uses flatbuffers-1.10 but we use flatbuffers-2.0 to match with all other modules flatbuffers version."